### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+ - [Orchestra libs](https://github.com/open-orchestra/open-orchestra-libs/compare/v1.1.0-alpha4...v1.1.0-beta)
+ - [Orchestra Mongo libs](https://github.com/open-orchestra/open-orchestra-mongo-libs/compare/v1.1.0-alpha4...v1.1.0-beta)
+ - [Media file bundle](https://github.com/open-orchestra/open-orchestra-media-file-bundle/compare/v1.1.0-alpha4...v1.1.0-beta)
+ - [Elastica bundle](https://github.com/open-orchestra/open-orchestra-elastica-bundle/compare/v1.1.0-alpha4...v1.1.0-beta)
+
+## Configuration changes
+
+- Adding bundle ``MongoDBMigrationsBundle`` [#853](https://github.com/open-orchestra/open-orchestra/pull/853)
+- dependencies version are fixed : ``merge ~1.2.0``, ``backbone ~1.2.3``, ``jquery-minicolors ~2.2.3``, ``backbone.wreqr ~1.3.5``. [#1397](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1397)
+- bower dependency ``underscore`` is removed, this package is include by ``backbone`` [#1397](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1397)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: is included

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+- dependencies version are fixed : ``merge ~1.2.0``, ``backbone ~1.2.3``, ``backbone.wreqr ~1.3.5``. [#136](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/136)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: dependencies versions

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+ - [Media admin bundle](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/compare/v1.1.0-alpha4...v1.1.0-beta)
+ - [Orchestra libs](https://github.com/open-orchestra/open-orchestra-libs/compare/v1.1.0-alpha4...v1.1.0-beta)
+ - [Orchestra Mongo libs](https://github.com/open-orchestra/open-orchestra-mongo-libs/compare/v1.1.0-alpha4...v1.1.0-beta)
+ - [Media file bundle](https://github.com/open-orchestra/open-orchestra-media-file-bundle/compare/v1.1.0-alpha4...v1.1.0-beta)
+ - [Elastica bundle](https://github.com/open-orchestra/open-orchestra-elastica-bundle/compare/v1.1.0-alpha4...v1.1.0-beta)
+
+## Configuration changes
+
+- Adding bundle ``MongoDBMigrationsBundle`` [#853](https://github.com/open-orchestra/open-orchestra/pull/853)
+- dependencies version are fixed : ``merge ~1.2.0``, ``backbone ~1.2.3``, ``jquery-minicolors ~2.2.3``, ``backbone.wreqr ~1.3.5``. [#1397](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1397)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: dependencies versions

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+- Allow recursive merge on tinyMce Stfalcon parameters [#142](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/142)
+- Status of contents containing a media field can now be changed [#138](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/138)
+- Remove media managment in blocks when form is disabled [#137](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/137)
+- Fix access denied error on published node access [#524](https://github.com/open-orchestra/open-orchestra-model-bundle/pull/524)
+- Fix broken varnish 4 vcl [#29](https://github.com/open-orchestra/open-orchestra-provision/pull/29)
+- As the pixel developer google chrome is not more accessible, I have suppressed the selenium role from our provisioning [#27](https://github.com/open-orchestra/open-orchestra-provision/pull/27)
+
+## New features
+
+- Block menu are invalidate when a node is removed, moved, restore and the status of a node changes . [#1531](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1531)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: Block menu cache are invalidated

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+- add voter on role usage [#1530](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1530)
+- increase ergo fo group form [#1522](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1522)
+- When a content type is deleted, navigation menu is refresh [#1521](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1521)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: refreshed

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+- When a website is created, an homepage for this site is also created. [#1518](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1518)
+- refresh the page after website creation to show it in the fiexed top nav list [#1515](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1515)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: fixed top nav

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+- fix save & back-to-list buttons on top [#1499](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1499)
+- Move template menu entry from Editorial to Administration [#1485](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1485)
+- allow to extend js form's behavior [#1473](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1473)
+- add loader in group tab view [#1467](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1467)
+- change new link to button [#1466](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1466)
+- Add sortable option for the datatable [#1451](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1451)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: to the datatable

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+- Fix resize with picture ratio lower than 1 [#178](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/178)
+- fix missed translation (open_orchestra_media_admin.block.display_media.title) [#162](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/162)
+- A super admin user can view all sites when he creates a new form [#160](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/160)
+- Fix media selection in tinyMce [#158](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/158)
+- Media events are not properly dispatched [#156](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/156)
+- Allow recursive merge on tinyMce Stfalcon parameters [#142](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/142)
+- Status of contents containing a media field can now be changed [#138](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/138)
+- Remove media managment in blocks when form is disabled [#137](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/137)
```
Comment (itkg-nanne) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: management

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+- Allow published node deletion [#1405](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1405)
+- add break line in tooltip helper (\n) [#1400](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1400)
+- allow published node delete [#1384](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1384)
+- auto-select type of search in datatable for content field type [#1382](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1382)
+- activate content list block in front [#202](https://github.com/open-orchestra/open-orchestra-display-bundle/pull/202)
+- Node cache-control policy differs with or without ESI support [#193](https://github.com/open-orchestra/open-orchestra-display-bundle/pull/193)
+- allow end user to format boolean condition in keywords filter [#41](https://github.com/open-orchestra/open-orchestra-libs/pull/41)
+- Media can be filtered on type when displayed from media folder [#179](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/179)
+- fix the new folder button on media modal [#177](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/177)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: This is a bugfix

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+- fix back-to-list buttons on top in modal upload [#172](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/172)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: This is a bugfix

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md (hunk 6 above)
Comment (itkg-nanne) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: fixed

File: app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+- Media library: applications can now describe their own specific alternatives image formats [#144](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/144)
+- Add a "Back to list" button on the media upload form in a modal context [#133](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/133)
+- Media selection integrates now the alternative selector [#132](https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/132)
+- Published flag feature [#529](https://github.com/open-orchestra/open-orchestra-model-bundle/pull/529)
+
+## Deprecated
+
+- ``initializeNewNode``  is deprecated and replace by ``initializeNode`` in class ``OpenOrchestra\Backoffice\Manager\NodeManager``. this method will be removed in 1.2.0 [#1518](https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1518)
```
Comment (itkg-ngilain) [inline app/Resources/doc/changeLog/CHANGELOG-1.1.0-beta.md]: replaced

**Pull request merged**